app_dart/lib/src/model/ci_yaml/ci_yaml.dart

```diff
@@ -165,6 +167,13 @@ class CiYaml {
           }
         }
       }
+
+      /// Check the dependencies for the current target if it is viable and to
+      /// be added to graph.
+      final String? dependencyJson = target.properties['dependencies'];
+      if (dependencyJson != null) {
```

Comment:
This will need to temporarily skip on release branches. When the ci.yaml changes roll to stable (O(months)), we can make.it the default everywhere

Reply:
Is the branch something that can be gathered from the target.proto?

Reply:
You should be able to pass it to _validate

https://cs.opensource.google/flutter/cocoon/+/main:app_dart/lib/src/model/ci_yaml/ci_yaml.dart;l=23

Reply:
How would I know based on the branch string if it is a release branch?